StaticGUI.py

```diff
@@ -190,8 +190,10 @@ def staticGridSimulation():
 
         root = Tk()
         # start GUI and run animation
+        print(*search.direction_commands(path), sep="\n")
```

Comment:
Should we keep this in the statis GUI??? I think it is good, but is it a final version that is working?